### PR TITLE
sig-multicluster 2022 annual report

### PR DESCRIPTION
**Pull request by JeremyOT**
[Draft]

Updates the 2022 annual report for sig-mc

### CODE REVIEW

File: sig-multicluster/annual-report-2022.md [hunk 1]
```diff
@@ -4,66 +4,67 @@
...
 4. If your group has special training, requirements for reviewers/approvers, or processes beyond the general [contributor guide],
    does your [CONTRIBUTING.md] document those to help **existing** contributors grow throughout the [contributor ladder]?
 
-   -
+   - NEEDS UPDATING
 
 5. Does the group have contributors from multiple companies/affiliations?
 
-   -
+   - Yes
```
Comment (mrbobbytables) [inline sig-multicluster/annual-report-2022.md:53]: This doesn't include any work in K/K, but for repos owned by sig-mc, this dashboard shows # of orgs

https://k8s.devstats.cncf.io/d/8/company-statistics-by-repository-group?orgId=1&var-period=d7&var-metric=contributions&var-repogroup_name=SIG%20Multicluster&var-repo_name=kubernetes%2Fkubernetes&var-companies=All&viewPanel=1&from=1641016800000&to=1672466400000

It looks like there 5-6 active companies

File: sig-multicluster/annual-report-2022.md [hunk 2]
```diff
@@ -4,66 +4,67 @@
...
+      [1645 - MCS API](https://github.com/kubernetes/enhancements/tree/master/keps/sig-multicluster/1645-multi-cluster-services-api)
 
 
 ## Project health
 
 1. What areas and/or subprojects does your group need the most help with?
    Any areas with 2 or fewer OWNERs? (link to more details)
 
-   -
+   - There has been continued and repeated interest in multicluster controller development, but we need more people to come and share specific use cases, previous attempts at implementations, and/or concerns before we can hope to propose any generally useful solutions.
```
Comment (mrbobbytables) [inline sig-multicluster/annual-report-2022.md:30]: Is there a repo / issue that can be linked here? If someone external to the project looked at it, where would they go to provide those use-cases?

File: sig-multicluster/annual-report-2022.md [hunk 3]
```diff
@@ -4,66 +4,67 @@
...
 
-   -
+   - Yes
 
 6. Are there ways end users/companies can contribute that they currently are not?
    If one of those ways is more full time support, what would they work on and why?
 
-   -
-   -
+   - The community we have is great, we just need more of it.
```
Comment (mrbobbytables) [inline sig-multicluster/annual-report-2022.md:58]: Similar to the above, could link out to an issue or something to direct folks that are potentially interested.

**Review by mrbobbytables — COMMENTED** [review]
Just a few comments, the big thing is links out to how people can follow up. These reports get summarized and sent to the GB, end user and member companies. If MC interests them, how/where do they send people?

File: sig-multicluster/annual-report-2022.md (hunk 1 above)
Comment (JeremyOT) [reply sig-multicluster/annual-report-2022.md:53]: thanks, added the stat

File: sig-multicluster/annual-report-2022.md (hunk 2 above)
Comment (JeremyOT) [reply sig-multicluster/annual-report-2022.md:30]: Linked a brainstorming doc the community has put together

File: sig-multicluster/annual-report-2022.md (hunk 3 above)
Comment (JeremyOT) [reply sig-multicluster/annual-report-2022.md:58]: filled in with more info

**Comment by mrbobbytables** [comment]
Thanks for pulling it together 🙏 
/lgtm

**Pull request merged**